LBHackney-IT/lbh-fss-portal-frontend
# Feature/view my services

**Description**

When navigating to **Your listings** you see a list of your services 

**Test instruction**

- pull this branch 
- `yarn install`
- navigate to the **Your listings** page and check it displays correctly for users with different information/permissions - to simulate users with different information/permissions navigate to **mock-api/account/GET.js** and comment/uncomment api response as appropriate 
 
- **VCSO - NO services but WITH organisation**  -  should see the add a service form ('service name' field pre-populated with 'organisation name'), see screenshot below

![image](https://user-images.githubusercontent.com/69238887/94621229-d7e78380-02a7-11eb-97c3-380091c636d0.png)

- **VCSO - WITH services and WITH organisation**  -   should see table of services, see screenshot below 

![image](https://user-images.githubusercontent.com/69238887/94621283-ec2b8080-02a7-11eb-8b4d-29df4f6da275.png)

- **VCSO - WITH services but WITHOUT organisations**  -  should see the add a service form ('service name' empty), see screenshot below

![image](https://user-images.githubusercontent.com/69238887/94621361-04030480-02a8-11eb-9540-760aed5ba1c0.png)

- **VCSO - WITH services but WITHOUT organisations**  -  should see an empty page with just an 'Add service' (this page will be populated in a later MR), see screenshot below 

![image](https://user-images.githubusercontent.com/69238887/94621152-b8e8f180-02a7-11eb-97e9-14bb96939e65.png)


Hope this makes sense - let me know if you have any questions 


**Ticket**

https://app.clickup.com/t/9e5u85
